packages/cli/src/cmds/preview/preview.ts

```diff
@@ -85,6 +85,10 @@ export default class Preview extends YargsCommand {
       array: false,
       string: true,
     });
+    yargs.option("env", {
+      description: "select an existed env for the project",
+      string: true,
+    });
```

Comment:
Any default value for `env`?

Reply:
No default value, if the user does not specify an env he need to choose one.
![image](https://user-images.githubusercontent.com/13211513/129654757-de30baa2-7ba1-4412-8399-398a2bd03702.png)

Reply:
Make sense. For local preview, `env` will be ignored, right?

Reply:
Yes.